fenix/app/onboarding.fml.yaml

```diff
@@ -110,6 +120,20 @@ objects:
         description: The text to display on the secondary button.
         # This should never be defaulted.
         default: ""
+      prerequisites:
+        type: List<String>
+        description:
```

Comment:
multi-line descriptions tend to have `description: >`. I am unsure whether this has an impact on the formatting of the description